.github/workflows/release.yml

```diff
@@ -14,63 +17,100 @@ on:
 
 jobs:
   build:
+    # Define a job to build binaries for multiple OS targets
     name: Build for ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Specifying os: and include: is redundant, and leads to surprises if they are not one-to-one